### PR TITLE
Refactoring learning rate (lr)

### PR DESCRIPTION
**Pull request by autoih**
This PR fix the redundant part of getter and setter in `lr` (it recommended to use `learning_rate`).

### CODE REVIEW

**Comment by autoih** [comment]
Thanks for the reminder @WindQAQ. Updated.
